pages/glossary/changes-in-content.md

```diff
@@ -4,15 +4,25 @@ key: changes-in-content
 unambiguous: true
 objective: true
 input_aspects:
+  - Accessibility tree
+  - Audio output
   - CSS styling
   - DOM tree
 ---
 
-Changes in the content of a [web page][] occur when at least one of the following occurs:
+A _change in the content_ of a [web page][] caused by an event occurs when, by comparing the [web page][] before and after the event firing, at least one of the following occurs:
```

Comment:
This does assume that firing the event is indeed causing the change.

If there is a page with audio playing non-stop (so, permanent change in content), then any event will be consider as a cause for this (because comparing the page before and after firing the event creates an **audible change**)

So, if there is a non-stop audio and a device motion event that does **not** cause any change in the content (e.g. it update a database in the backend), the definition will nonetheless consider the **audible change** as a "change in the content caused by" the device motion and the rule will fail while 2.5.4 doesn't (non-stop audio is bad for 1.4.2, of course).

Reply:
I shouldn't have used "event" because it was obvious that was going to be confused with dom events. I was using "event" with the generic meaning of "origin of the change" whatever that might be (most probably a dom event!)

Nevertheless, in your example, according to this definition, there would be no "change in content". There is a paragraph after the list explaining that when time-based media is at play, the comparison is not between what was before the event and what is after the event, but instead, between what would be if there was no event and what is when there is an event.

Reply:
* "event": the use both of the word "event" and "firing" does make the confusion with DOM events likely. Not really important due to the rest.
* I have indeed missed the rest of the def 😥